test/hybrid-viewer.cpp

```diff
@@ -342,7 +345,9 @@ class fltkview : public Fl_Gl_Window
             case ' ':
                 if(sim)
                 {
+                    std::cout << "      hybrid_step -------------------------------------------            \n";
                     sim->hybrid_step();
+                    exit(0);
```

Comment:
Why would you want to to exit after stepping?

Reply:
This is also for debugging. I will not commit those things in the future.